spec/system/pages/home_spec.rb

```diff
@@ -8,15 +8,25 @@
   end
 
   it "is the correct page" do
-    expect(page).to have_content("Helping you\nteach computing")
+    expect(page).to have_content("Helping you teach computing")
   end
 
   it "header is accessible" do
     expect(page).to be_accessible.within("header")
   end
 
   it "page is accessible" do
-    expect(page).to be_accessible.within("#main-content").excluding(".axe-skip-a11y-test")
+    exclusions = %w[
+      ".axe-skip-a11y-test"
+      "ncce-home-hero__house-background-corner"
+      "ncce-home-hero__house-roof"
+      "ncce-home-hero__house-wall-left"
+      "ncce-home-hero__house-wall-right"
+      "ncce-home-hero__house-base"
+      "ncce-home-hero__person-corner-left"
+      "ncce-home-hero__person-corner-right"
+    ]
+    expect(page).to be_accessible.within("#main-content").excluding(exclusions)
```

Comment:
Could we instead make the page accessible rather than just ignoring it 😆?

For each of those images, if you just add an area-hidden, it ought to do the trick

Reply:
So I had added the aria_hidden fields but will look into why this isn't preventing the tests from failing 👀